### ISSUE TITLE
File downloads not working in Element web

### ISSUE THREAD

**Issue by clicit** [YunoHost-Apps/element_ynh#82]
The initially requested (#71) fix for the file download problem doesn't seem to solve the issue as expected, so I opened this ticket to push the topic again. 
#71 actually has some effect, but the file-download is still not possible.

### Context

- Hardware: VPS bought online
- YunoHost version: YunoHost 11.0.10.2 (stable)
- I have access to my server: Through SSH, through the webadmin, direct access via keyboard / screen

### Steps to reproduce

   1. Let someone send you  e.g. a ZIP file
   2. Click on the download icon in Element web (arrow-down) 

### Expected behavior

File is downloaded to the local storage

### Logs
After applying #71 by myself, I get the following error message in the browser console:
![image](https://user-images.githubusercontent.com/26301562/200002916-f5a9601f-aac3-4a82-8f0a-59e19631bfd9.png)

**Comment by vsatmydynipnet** [comment]
Same issue here. :-(

**Comment by oskrak** [comment]
Same issue :/

**Comment by pagiraud** [comment]
Me too.

**Comment by ericgaspar** [comment]
Do you still have this issue?
(I can't reproduce this issue...)

**Comment by pagiraud** [comment]
Yes, I still have this issue. I tested it by sending some PDF from Element on my Android device and trying to open it from my Element web.

**Comment by ericgaspar** [comment]
there is a fix on testing (may not be the definitive solution...)

**Comment by clicit** [comment]
Yes. 

> Do you still have this issue? (I can't reproduce this issue...)

Yes, issue still persists.

**Comment by ericgaspar** [comment]
should be fixed with  #102

**Comment by ericgaspar** [comment]
is the issue solved with the last update 1.11.28~ynh2 ?

**Comment by clicit** [comment]
Unfortunately no, issue still persists.

**Comment by pagiraud** [comment]
I confirm that, issue persists.

**Comment by clicit** [comment]
any plans to tackle this problem? It's suboptimal that every user has to install the desktop app also, just because to be able to download files..

**Comment by ericgaspar** [comment]
of course we plan to tackle this problem, but nobody came with a working solution yet
